styles/components/profile.css

```diff
@@ -0,0 +1,58 @@
+.profile {
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  grid-template-rows: 1fr auto 1fr 1fr;
+}
+
+.profile__img {
+  margin: 0;
+  justify-self: start;
+  align-self: center;
+}
+
+.profile__img--boarder {
```

Comment:
border ;)
Why do you use an extra class for the border-radius?

Reply:
haha thank you 😂🙈